soc/lowrisc/opentitan/rom_header.S

```diff
@@ -6,16 +6,18 @@
 
 #include <zephyr/toolchain.h>
 
-/* imports */
-GTEXT(__start)
+/* exports */
+GTEXT(__rom_header)
 
-/* OpenTitan manifest consists of 896 bytes (224 words) containing signature,
- * device ID, version info, etc. The test ROM ignores all of these fields
- * except for entry point (final word in manifest).
+/* OpenTitan manifest consists of 964 bytes (226 words, plus 15 words
+ * of manifest extension) containing signature, device ID, version
+ * info, etc. The test ROM ignores all of these fields except for entry
+ * point (final word in the base manifest).
  */
 SECTION_FUNC(rom_header, __rom_header)
-	.rept(223)
+	.rept(225)
 	.word 0
 	.endr
 	/* Entry point is relative to the beginning of manifest. */
-	.word(__start - __rom_header)
+	.word(__reset - __rom_header)
```

Comment:
Oh, maybe you changed it because the __start entry point wasn't working, because the extension table now contains __start!?!? If that's the case, padding out zero through the extension table (and thus moving __start out of the extension table) should make things work.

Reply:
yes indeed - my bad, a left over change from debug.